### DISCUSSION TITLE
Can't access "rcamera.h"

### DISCUSSION THREAD

**Discussion by mammuen** [raysan5/raylib#3937] [Q&A]
Hello 

I am quite new to Raylib, and I have been going through some of the examples and tutorials. I wanted to try out some examples in 3d, and I tried copying the code from "3d_camera_first_person" (link: https://github.com/raysan5/raylib/blob/master/examples/core/core_3d_camera_first_person.c). However I can't figure out how to import the header file correctly and I keep getting an error in VScode that says "cannot open source file rcamera.h":

<img width="912" alt="Screenshot 2024-04-25 at 16 49 15" src="https://github.com/raysan5/raylib/assets/101991418/bcb5cb29-bc66-4395-af74-5660f2b9c19f">

I am working in macOS and I have configured my VScode using this tutorial: https://github.com/raysan5/raylib/wiki/Using-raylib-in-VSCode, which removed the red squiggly lines for the `#include "raylib.h"` line. However i still get the above mentioned error for the line `#include "rcamera.h"`

**I use cmake to build 

And my cmake file is:

<img width="860" alt="Screenshot 2024-04-25 at 16 54 50" src="https://github.com/raysan5/raylib/assets/101991418/547d52ca-0ed5-45f6-926d-226e878d103f">

## Any help would be appreciated on how I can get the red squiggly lines to dissapear. Thanks in advance.

**Comment by RobLoach** [comment]
There have been some updates in how CMake is used since 3.0. I'd recommend checking out the CMAke example proejct at https://github.com/raysan5/raylib/tree/master/projects/CMake .

**Reply by mammuen** [reply]
It works when I copy the contents of that CMakeLists.txt file into my project, but only because it then downloads raylib and builds it within the build directory, even though I already have raylib downloaded via homebrew.

If possible I would like to just use the version i downloaded via homebrew, and not download a new one. I just need to know what to include within the CMakeLists file, probably something similar to:

`
find_path(RL_INCLUDE_DIR NAMES raylib.h PATHS /opt/homebrew/opt/raylib/include)
include_directories(${RL_INCLUDE_DIR})
`

But which finds the `rcamera.h` header file.

**Comment by cortex-m4** [comment]
Today I encounter a same problem like your, I solve it use this way: **Copy rcamera.h into your include file path and rebuild your project**.

**Reply by cortex-m4** [reply]
A little different is I have not use CMake to build, I use makefile
